Code

```diff
@@ -88,7 +88,7 @@ btnSubmit.addEventListener('click', () => {
 });
 
 const btnRemove = document.getElementById('remove');
-
+console.log('This is just for testing the scenarios');
 btnRemove.addEventListener('click', () => {
   operations.removeFromList();
```

Comment:
- on your javascript code line 91 i suggest you to remove the console.log for a better clean best practics.
- Kindly update your readme file add project name and project description   
- I suggest you to document all your files seperately by creating seperate files html,css and javascript files link to your html,  in good order .
- i suggest you create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader.